modules/components/pages/outputs/cypher.adoc

```diff
@@ -0,0 +1,409 @@
+= cypher
+:type: output
+:status: experimental
```

Comment:
If experimental is now considered beta, we can replace this with :page-beta: true